ci_tools/check_new_coverage.py

```diff
@@ -20,7 +20,18 @@
 diff = get_diff_as_json(args.diffFile)
 untested, file_contents = cov.get_untested_lines(args.coverageFile)
 
-new_untested = cov.allow_untested_error_calls(cov.compare_coverage_to_diff(untested, diff))
+print(untested)
+new_untested = cov.compare_coverage_to_diff(untested, diff)
+
+print(new_untested)
+
+new_untested = cov.allow_untested_error_calls(new_untested)
+
+print(new_untested)
+
+new_untested = cov.allow_untested_debug_code(new_untested)
+
+print(new_untested)
```

Comment:
Is this debug code, or do you want to print to terminal for the user?

Reply:
You are right, this is debug code. It has been removed: fb603b7